server/controllers/paintings/addPainting.js

```diff
@@ -0,0 +1,45 @@
+const { addPaintingSchema } = require('../../utils/validation');
+const uploadImg = require('../../utils/uploadImg');
+const { addPaintingQuery } = require('../../database/queries');
+
+exports.addPainting = async (req, res, next) => {
+  try {
+    const { title, description, category, property } = req.body;
+    const { id } = req.user;
+    const { paintingImg } = req.files;
+    await addPaintingSchema.validate(
+      { title, description, category, property, paintingImg },
```

Comment:
```suggestion
      { ...req.body, paintingImg },
```

Reply:
The best practice is to identify the values to validate, cause at front side the user could add undesirable value at the request body